specs/agents/tracing-sampling.md

```diff
@@ -63,7 +63,7 @@ Metrics will be more accurate when the sampling rate is high.
 With lower sampling rates the server is able to calculate representative, but less accurate, metrics.
 If the sampling rate is 0 then no metrics will be calculated at all.
 
-Agents will record the sampling rate on transactions and spans as `sample_rate`. e.g.
+Agents MUST record the sampling rate on transactions and spans as `sample_rate`. e.g.
```

Comment:
```suggestion
Agents MUST record the sampling rate on transactions and spans as `sample_rate`, when available. i.e.
```

I think this should have a caveat to say it MUST be recorded when it's available, to align with the last line of the Propagation section, which says

> If there is no tracestate or no valid es entry with an s attribute, then the agent must omit `sample_rate` from non-root transactions and their spans.

Reply:
It's a very good point about the sampling rate not always being available - I've changed this part, please take a look.

Regarding "e.g." vs "i.e." - I'm not a native English speaker so I would like to make sure I correctly understand you proposal :) I though that "e.g." is equivalent to "for example" while "i.e." is equivalent to "that is"/"which means". In our case the part that follows "e.g." is indeed an example of a payload with `sample_rate` so wouldn't "for example" fit better in this case?

Reply:
> Regarding "e.g." vs "i.e."

Yes, it should be `E.g.` or `For example`.

Reply:
I'm not an English major, but I think it ought be i.e. in this case because what follows appear to be exact definitions of what the recorded `sample_rate` for any span and transaction must look like, and there are no other options. I can see an argument for why they could be considered examples because they include exemplar `name` properties. Removing the `name` properties would make them clear definitions for the previous statement.

Reply:
Since there seem to be different opinions about "e.g." vs "i.e." and since it's not related to the purpose of this PR I prefer to leave as it is and we can revisit it later.